07.ls_object/lib/ls_object.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'pathname'
+require_relative 'ls_file_stat'
+require_relative 'screen'
+
+class LsObject
+  def initialize(paths, options)
+    @paths = paths.empty? ? ['.'] : paths
+    @options = options
+  end
+
+  def main
+    @options[:header] = !(@paths.length == 1 || @paths.empty?)
+
+    @screen_src = { '' => [] }
+    create_sorted_path_names(@paths).each do |path_name|
```

Comment:
```suggestion
    create_sorted_pathnames(@paths).each do |pathname|
```

Pathnameオブジェクトなのでアンダースコアは不要かも